babel.config.js

```diff
@@ -55,6 +69,31 @@ const config = {
 			},
 		],
 		isCalypsoClient && './inline-imports.js',
+		cssModules && [
+			path.join(
+				__dirname,
+				'server',
+				'bundler',
+				'babel',
+				'babel-plugin-transform-rename-attribute'
+			),
+			{
+				from: 'className',
+				to: 'styleName',
+			},
+		],
+		cssModules && [
+			'babel-plugin-react-css-modules',
+			{
+				context: __dirname,
+				filetypes: {
+					'.scss': {
+						syntax: 'postcss-scss',
```

Comment:
Would the goal here be to drop our independent SASS compilers? Is https://github.com/postcss/postcss-scss sufficient? It doesn't sound like a "full" sass compiler.